src/Component/GeneralEntity/GeneralEntityRoot/GeneralEntityRoot.tsx

```diff
@@ -262,13 +262,11 @@ export function GeneralEntityRoot<T extends BaseEntity>({
    */
   useEffect(() => {
     if (!entityId) {
-      setId(undefined);
```

Comment:
Why is this needed? I would have assumed a new entity should never an id?

Reply:
reverted since no reason can be stated at the current moment